README.md

```diff
@@ -142,11 +142,13 @@ From this page, open the "Create credentials" drop-down and select "Service acco
 
 On the next screen, you can give the service account a name. You can name it anything you’d like. In the service account id you’ll see an email address. We’ll use this email address later on in this guide. Select "JSON" as the key type and click "Create" to download the JSON file. You will get a warning that the service account does not have a role, you can safely ignore this and create the service account without assigning a role.
 
+If you have delegated domain-wide access to the service account and you want to impersonate a user account, specify the email address of the user account in the config file.
+
 ![7](./docs/v2/7.png)
 
 Save the json inside your Laravel project at the location specified in the `service_account_credentials_json` key of the config file of this package. Because the json file contains potentially sensitive information, I don't recommend committing it to your git repository.
 
-Now that everything is set up on the API site, we’ll need to configure some things on the Google Calendar site. Head over to Google Calendar and view the settings of the calendar you want to work with via PHP.  On the "Share with specific people" tab press the "Add people" button and add the service account id that was displayed when creating credentials on the API site. 
+Now that everything is set up on the API site, we’ll need to configure some things on the Google Calendar site. Head over to Google Calendar and view the settings of the calendar you want to work with via PHP.  On the "Share with specific people" tab press the "Add people" button and add the service account id that was displayed when creating credentials on the API site.
```

Comment:
This is an optional step, right? Maybe also mention that in the text.

Reply:
As far as we went trying to get this to work with our Workspace account, it's a required step to share the calendar with the service account.